glean-core/android/src/test/java/mozilla/telemetry/glean/private/UuidMetricTypeTest.kt

```diff
@@ -46,12 +40,14 @@ class UuidMetricTypeTest {
             sendInPings = listOf("store1")
         )
 
+        // Check that there is no UUID recorded
+        assertFalse(uuidMetric.testHasValue())
+
         // Record two uuids of the same type, with a little delay.
-        val uuid = uuidMetric.generateAndSet()
+        uuidMetric.generateAndSet()
 
-        // Check that data was properly recorded.
```

Comment:
Let's not change this, if possible, so that it's easier to diff while we transition to glean-core.